src/library_html5.js

```diff
@@ -488,17 +488,25 @@ var LibraryHTML5 = {
 
 #if !DISABLE_DEPRECATED_FIND_EVENT_TARGET_BEHAVIOR
     if (Module['canvas']) {
-      var rect = getBoundingClientRect(Module['canvas']);
-      HEAP32[idx + {{{ C_STRUCTS.EmscriptenMouseEvent.canvasX / 4 }}}] = e.clientX - rect.left;
-      HEAP32[idx + {{{ C_STRUCTS.EmscriptenMouseEvent.canvasY / 4 }}}] = e.clientY - rect.top;
+      const target = Module['canvas'];
```

Comment:
In general we don't use `const` yet in the core JS libraries.  Can you just use `var` for now?